src/cephadm/cephadm

```diff
@@ -2234,6 +2234,8 @@ def get_container_mounts(ctx, fsid, daemon_type, daemon_id,
     if daemon_type == 'osd':
         mounts['/sys'] = '/sys'  # for numa.cc, pick_address, cgroups, ...
         # selinux-policy in the container may not match the host.
+        if not os.path.exists('/usr/share/empty'):
+            os.mkdir('/usr/share/empty', 0o755)
```

Comment:
I'd avoid creating `'/usr/share/empty'` on the host. what about only mounting it, if it exists?

Reply:
That works for me too.

Reply:
The real thing that we want to resolve is the possible mismatch between `/sys/fs/selinux` on the host and what is installed within the container.

If all systems that lack `/usr/share/empty` will _also_ not have selinux enabled in `/sys/fs/selinux`, it's ok.

Reply:
We mount only if the folder exists